app/Task.php

```diff
@@ -3,8 +3,14 @@
 namespace App;
 
 use Illuminate\Database\Eloquent\Model;
+use App\User;
 
 class Task extends Model
 {
-    //
+    protected $fillable = ['name'];
+
+    public function user()
+    {
+    	return $this->belongsTo(User::class, 'user_id');
```

Comment:
Nếu foren_key đã đúng chuẩn rồi thì chỉ cần `return $this->belongsTo(User::class);`